maas/provider.go

```diff
@@ -68,8 +68,8 @@ func Provider() *schema.Provider {
 			"maas_vlan":                       resourceMaasVlan(),
 			"maas_subnet":                     resourceMaasSubnet(),
 			"maas_subnet_ip_range":            resourceMaasSubnetIPRange(),
-			"maas_dns_domain":                 resourceMaasDnsDomain(),
-			"maas_dns_record":                 resourceMaasDnsRecord(),
+			"maas_dns_domain":                 resourceMaasDNSDomain(),
```

Comment:
Maybe we should change `Maas` to `MAAS` as well? And add it to the linter rules? ;)